master/pyproject.toml

```diff
@@ -0,0 +1,9 @@
+[tool.sqlfluff.core]
+templater = "jinja"
+sql_file_exts = ".sql,.sql.j2,.dml,.ddl"
+exclude_rules = "L034,layout.spacing"
```

Comment:
what is the layout rule... i may like that one lol

Reply:
I believe it's this one. https://docs.sqlfluff.com/en/stable/layout.html#configuring-layout-and-spacing

2.1.4 also adds some other changes vs 2.1.1

I'd like to get this in to get the linting and formatting orchestration in and then let's tweak it in a separate PR

Reply:
i eyed the rules, huge fan of most of them. i frequently leave them as nits.